From 68213102ffd61f6b1243dbf09533de4947c6b68e Mon Sep 17 00:00:00 2001
From: Cedric Bondier <cedricx.bondier@intel.com>
Date: Mon, 7 Jan 2013 16:07:34 +0100
Subject: Multivendor support: adding libbt-vendor conf for x86 targets

BZ: 76413 72228 99491 103267 112157 117525

Including the specific Intel configuration file for all x86 targets
in libbt-vendor configuration.

Some compile-time configuration variables for Bluetooth are board
specific.

As a consequence, the Bluedroid configuration files are created in
product device specific configuration folders.

Category: device enablement
Domain: CWS.BT-Core
Origin: internal
Upstream-Candidate: no, proprietary

Change-Id: Ia7de3b9049b0586cea0b8c24b4239a66af203a6e
Orig-Change-Id: I788675811f96b45e28b00c18cf597ad3025fd195
Orig-Change-Id: I6ee8fca5ec7033388e79ff447418386344a8453a
Orig-Change-Id: Ic18ade5b3f480d8c99911d77054f14bdf827a1b6
Orig-Change-Id: I65d36b935f8cb087389a9575c108b891a103e4fb
Orig-Change-Id: Ie1778787b1b6cc2bab3018afc9ce56046f470811
Orig-Change-Id: I9b855a58892d7b51f733aa7ced307d366c27f2cf
Signed-off-by: Raffaele Aquilone <raquiloX>
Signed-off-by: Julien Gros <julien.gros@intel.com>
Signed-off-by: Celine Chotard <celine.chotard@intel.com>
Signed-off-by: Laurent FERT <laurent.fert@intel.com>
Signed-off-by: Patrick Combes <patrick.combes@intel.com>
Reviewed-on: http://android.intel.com:8080/85372
Reviewed-on: http://android.intel.com:8080/85605
Reviewed-on: http://android.intel.com:8080/101596
Reviewed-on: http://android.intel.com:8080/104375
Reviewed-on: http://android.intel.com:8080/111116
Reviewed-on: http://android.intel.com:8080/114601
Reviewed-by: Bondier, CedricX <cedricx.bondier@intel.com>
Reviewed-by: Aquilone, RaffaeleX <raffaelex.aquilone@intel.com>
Reviewed-by: Biselli, MirkoX <mirkox.biselli@intel.com>
Reviewed-by: Chen, HuaqiangX <huaqiangx.chen@intel.com>
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Reviewed-by: Isaksen, FrodeX <frodex.isaksen@intel.com>
Reviewed-by: Combes, Patrick <patrick.combes@intel.com>
Reviewed-by: Vamour, Cedric <cedric.vamour@intel.com>
Reviewed-by: Fert, Laurent <laurent.fert@intel.com>
Reviewed-by: Gros, Julien <julien.gros@intel.com>
Reviewed-by: Imberton, Guilhem <guilhem.imberton@intel.com>
Reviewed-by: Boniface, XavierX <xavierx.boniface@intel.com>
Reviewed-by: Molero Rodenas, BeatrizX <beatrizx.molero.rodenas@intel.com>
Reviewed-by: Tinelli, Vincent <vincent.tinelli@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
Tested-by: Zurmely, PierreX <pierrex.zurmely@intel.com>
Tested-by: Tinelli, Vincent <vincent.tinelli@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: cactus <cactus@intel.com>
Tested-by: buildbot <buildbot@intel.com>
Reviewed-on: http://android.intel.com:8080/123799
Reviewed-by: Poignet, Cedric <cedric.poignet@intel.com>
---
 conf/intel/baylake/bt_vendor.conf      |    5 +++++
 conf/intel/bodegabay/bt_vendor.conf    |    5 +++++
 conf/intel/byt_m_crb/bt_vendor.conf    |    5 +++++
 conf/intel/byt_t_ffrd10/bt_vendor.conf |    5 +++++
 conf/intel/byt_t_ffrd8/bt_vendor.conf  |    5 +++++
 conf/intel/ctp7160/bt_vendor.conf      |    5 +++++
 conf/intel/ctpscalelt/bt_vendor.conf   |    5 +++++
 conf/intel/redhookbay/bt_vendor.conf   |    5 +++++
 conf/intel/saltbay_pr0/bt_vendor.conf  |    5 +++++
 conf/intel/saltbay_pr1/bt_vendor.conf  |    5 +++++
 conf/intel/victoriabay/bt_vendor.conf  |    5 +++++
 11 files changed, 55 insertions(+), 0 deletions(-)
 create mode 100644 conf/intel/baylake/bt_vendor.conf
 create mode 100644 conf/intel/bodegabay/bt_vendor.conf
 create mode 100644 conf/intel/byt_m_crb/bt_vendor.conf
 create mode 100644 conf/intel/byt_t_ffrd10/bt_vendor.conf
 create mode 100644 conf/intel/byt_t_ffrd8/bt_vendor.conf
 create mode 100644 conf/intel/ctp7160/bt_vendor.conf
 create mode 100644 conf/intel/ctpscalelt/bt_vendor.conf
 create mode 100644 conf/intel/redhookbay/bt_vendor.conf
 create mode 100644 conf/intel/saltbay_pr0/bt_vendor.conf
 create mode 100644 conf/intel/saltbay_pr1/bt_vendor.conf
 create mode 100644 conf/intel/victoriabay/bt_vendor.conf

diff --git a/conf/intel/baylake/bt_vendor.conf b/conf/intel/baylake/bt_vendor.conf
new file mode 100644
index 0000000..62ffeb9
--- /dev/null
+++ b/conf/intel/baylake/bt_vendor.conf
@@ -0,0 +1,5 @@
+# UART device port where Bluetooth controller is attached
+UartPort = /dev/ttyMFD0
+
+# Firmware patch file location
+FwPatchFilePath = /etc/firmware/
diff --git a/conf/intel/bodegabay/bt_vendor.conf b/conf/intel/bodegabay/bt_vendor.conf
new file mode 100644
index 0000000..62ffeb9
--- /dev/null
+++ b/conf/intel/bodegabay/bt_vendor.conf
@@ -0,0 +1,5 @@
+# UART device port where Bluetooth controller is attached
+UartPort = /dev/ttyMFD0
+
+# Firmware patch file location
+FwPatchFilePath = /etc/firmware/
diff --git a/conf/intel/byt_m_crb/bt_vendor.conf b/conf/intel/byt_m_crb/bt_vendor.conf
new file mode 100644
index 0000000..93aa23c
--- /dev/null
+++ b/conf/intel/byt_m_crb/bt_vendor.conf
@@ -0,0 +1,5 @@
+# UART device port where Bluetooth controller is attached
+UartPort = /dev/ttyMFD0
+
+# Firmware patch file location
+FwPatchFileRootPath = /etc/firmware/bt/
diff --git a/conf/intel/byt_t_ffrd10/bt_vendor.conf b/conf/intel/byt_t_ffrd10/bt_vendor.conf
new file mode 100644
index 0000000..93aa23c
--- /dev/null
+++ b/conf/intel/byt_t_ffrd10/bt_vendor.conf
@@ -0,0 +1,5 @@
+# UART device port where Bluetooth controller is attached
+UartPort = /dev/ttyMFD0
+
+# Firmware patch file location
+FwPatchFileRootPath = /etc/firmware/bt/
diff --git a/conf/intel/byt_t_ffrd8/bt_vendor.conf b/conf/intel/byt_t_ffrd8/bt_vendor.conf
new file mode 100644
index 0000000..93aa23c
--- /dev/null
+++ b/conf/intel/byt_t_ffrd8/bt_vendor.conf
@@ -0,0 +1,5 @@
+# UART device port where Bluetooth controller is attached
+UartPort = /dev/ttyMFD0
+
+# Firmware patch file location
+FwPatchFileRootPath = /etc/firmware/bt/
diff --git a/conf/intel/ctp7160/bt_vendor.conf b/conf/intel/ctp7160/bt_vendor.conf
new file mode 100644
index 0000000..62ffeb9
--- /dev/null
+++ b/conf/intel/ctp7160/bt_vendor.conf
@@ -0,0 +1,5 @@
+# UART device port where Bluetooth controller is attached
+UartPort = /dev/ttyMFD0
+
+# Firmware patch file location
+FwPatchFilePath = /etc/firmware/
diff --git a/conf/intel/ctpscalelt/bt_vendor.conf b/conf/intel/ctpscalelt/bt_vendor.conf
new file mode 100644
index 0000000..62ffeb9
--- /dev/null
+++ b/conf/intel/ctpscalelt/bt_vendor.conf
@@ -0,0 +1,5 @@
+# UART device port where Bluetooth controller is attached
+UartPort = /dev/ttyMFD0
+
+# Firmware patch file location
+FwPatchFilePath = /etc/firmware/
diff --git a/conf/intel/redhookbay/bt_vendor.conf b/conf/intel/redhookbay/bt_vendor.conf
new file mode 100644
index 0000000..26a7e5f
--- /dev/null
+++ b/conf/intel/redhookbay/bt_vendor.conf
@@ -0,0 +1,5 @@
+# UART device port where Bluetooth controller is attached
+UartPort = /dev/hci_tty
+
+# Firmware patch file location
+FwPatchFilePath = /etc/firmware/
diff --git a/conf/intel/saltbay_pr0/bt_vendor.conf b/conf/intel/saltbay_pr0/bt_vendor.conf
new file mode 100644
index 0000000..62ffeb9
--- /dev/null
+++ b/conf/intel/saltbay_pr0/bt_vendor.conf
@@ -0,0 +1,5 @@
+# UART device port where Bluetooth controller is attached
+UartPort = /dev/ttyMFD0
+
+# Firmware patch file location
+FwPatchFilePath = /etc/firmware/
diff --git a/conf/intel/saltbay_pr1/bt_vendor.conf b/conf/intel/saltbay_pr1/bt_vendor.conf
new file mode 100644
index 0000000..62ffeb9
--- /dev/null
+++ b/conf/intel/saltbay_pr1/bt_vendor.conf
@@ -0,0 +1,5 @@
+# UART device port where Bluetooth controller is attached
+UartPort = /dev/ttyMFD0
+
+# Firmware patch file location
+FwPatchFilePath = /etc/firmware/
diff --git a/conf/intel/victoriabay/bt_vendor.conf b/conf/intel/victoriabay/bt_vendor.conf
new file mode 100644
index 0000000..62ffeb9
--- /dev/null
+++ b/conf/intel/victoriabay/bt_vendor.conf
@@ -0,0 +1,5 @@
+# UART device port where Bluetooth controller is attached
+UartPort = /dev/ttyMFD0
+
+# Firmware patch file location
+FwPatchFilePath = /etc/firmware/
-- 
1.7.4.1

